salt/modules/zypper.py

```diff
@@ -41,6 +41,7 @@
 import salt.utils.stringutils
 import salt.utils.systemd
 from salt.utils.versions import LooseVersion
+from salt.utils.environment import get_module_environment
```

Comment:
Same as apt and yum, this will put `get_module_environment` in the `__salt__` dunder.